xml/System/Type.xml

```diff
@@ -6774,7 +6774,7 @@ For the `GetMethods(BindingFlags)` overload to successfully retrieve method info
   
  See <xref:System.Reflection.BindingFlags?displayProperty=nameWithType> for more information.  
 
-The <xref:System.Type.GetMethods%2A> method does not return methods in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which methods are returned, because that order varies.  
+In .NET 6 and below versions the <xref:System.Type.GetMethods%2A> method does not return methods in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which methods are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
In .NET 6 and earlier versions, the <xref:System.Type.GetMethods%2A> method does not return methods in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which methods are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```